src/main/resources/application.yml

```diff
@@ -41,8 +41,9 @@ rocketmq:
     # if this value is empty,use env value rocketmq.config.namesrvAddr  NAMESRV_ADDR | now, default localhost:9876
     # configure multiple namesrv addresses to manage multiple different clusters
     namesrvAddrs:
-      - 127.0.0.1:9876
-      - 127.0.0.2:9876
+#      - 10.253.141.99:9876;10.253.141.102:9876
+      - 100.76.6.97:9876
```

Comment:
Suggest using localhost but not specific host

Reply:
Forgot to change back when testing locally